Update React readme (#30096)
facebook/react: README.md

# [React](https://react.dev/) &middot; [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebook/react/blob/main/LICENSE) [![npm version](https://img.shields.io/npm/v/react.svg?style=flat)](https://www.npmjs.com/package/react) [![CircleCI Status](https://circleci.com/gh/facebook/react.svg?style=shield)](https://circleci.com/gh/facebook/react) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://legacy.reactjs.org/docs/how-to-contribute.html#your-first-pull-request)

> React is a JavaScript library for building web and native user interfaces.

Reviewer: The tagline on the homepage is "The library for web and native user interfaces", seems like we should match that? It's not just about web and native (and "native" is loaded because what does that include and why is that a bucket of all non-web surfaces??)

Reviewer: ```suggestion
The library for building web and native user interfaces.
```
